Source/WebCore/accessibility/AccessibilityObject.cpp

```diff
@@ -3995,6 +3995,10 @@ AccessibilityObjectInclusion AccessibilityObject::defaultObjectInclusion() const
     if (isARIAHidden())
         return AccessibilityObjectInclusion::IgnoreObject;
 
+    auto* localWebArea = webArea();
+    if (localWebArea && !localWebArea->shouldWebAreaExposeChildren())
+        return AccessibilityObjectInclusion::IgnoreObject;
```

Comment:
Why "local"? Another option would be `RefPtr webArea = this->webArea();` unless local is really meaningful here.

Reply:
Mentioned this elsewhere, but the I wanted a way to semantically differentiate the root web area and this elements parent web area, but using `this->webArea` is probably enough to do that, so I'll update that here.